Remotion/Web/Core/Res/Themes/NovaViso/HTML/Common.css

```
@@ -132,6 +132,68 @@ input[type="password"].remotion-themed,
   padding-right: var(--remotion-themed-spacing);
 }
 
+.remotion-themed input[type=radio],
+.remotion-themed input[type=checkbox]
+{
+  z-index: var(--z-index-hidden);
+  position: absolute;
+  opacity: 0;
+}
+
+.remotion-themed input[type=checkbox] + span,
+.remotion-themed input[type=checkbox] + label
+{
+  display: inline-block;
+}
+
+.remotion-themed input[type=checkbox] + label:before,
+.remotion-themed input[type=checkbox] + span:before,
+.remotion-themed input[type=radio] + label:before,
+.remotion-themed input[type=radio] + span:before
+{
+  display: inline-block;
+  content: "";
+}
+
+.remotion-themed input[type=checkbox] + label:before,
+.remotion-themed input[type=checkbox] + span:before
+{
+  background-image: url('../Image/sprite.svg#CheckBoxFalse');
+  margin: var(--remotion-themed-spacing);
+  vertical-align: middle;
+  height: 1.5em;
+  width: 1.5em;
+}
+
+.remotion-themed input[type=radio] + label:before,
+.remotion-themed input[type=radio] + span:before
+{
+  background-image: url('../Image/sprite.svg#RadioButtonFalse');
+  height: var(--default-icon-size);
+  width: var(--default-icon-size);
```

```suggestion
  height: var(--remotion-themed-img-width);
  width: var(--remotion-themed-img-width);
```
I think this should be indetical to checkbox. We can then also move the height/width to the shared declarations.